src/components/theme/getRootClassName.ts

```diff
@@ -0,0 +1,13 @@
+import {block} from '../utils/cn';
+
+import {ROOT_CLASSNAME} from './constants';
+
+const b = block(ROOT_CLASSNAME);
+
+interface RootMods {
```

Comment:
```suggestion
interface RootMods {
    nativeScrollbar?: boolean;
```

Reply:
We're planning to remove scroll customization, #1587